### DISCUSSION TITLE
post_digest shortcode attribute in regular campaign

### DISCUSSION THREAD

**Discussion by X-Raym** [hizzle-co/noptin#574] [Ideas]
Hi,

Currently post_digest shortcode attribute is only available in automated campaign.

Could be nice to have in regular campaign as well.

I guess meanwhile I could hook the email content if there are the necessary filter and plug this to the PHP function for post_digest, but sure it will be more handy out of the box.

Cheers!

**Comment by picocodes** [comment]
Hi,

If you're using the addons pack then this is now possible:- https://noptin.com/guide/sending-emails/how-to-display-posts-in-email-campaigns/
